src/test/java/org/springframework/samples/petclinic/vet/VetTests.java

```diff
@@ -40,4 +45,37 @@ public void testSerialization() {
         assertThat(other.getId()).isEqualTo(vet.getId());
     }
 
+    @Test
+    public void testGetSpecialties(){
+        Vet vet = new Vet();
+        vet.setFirstName("Zaphod");
+        vet.setLastName("Beeblebrox");
+        vet.setId(123);
+        Specialty radiology = new Specialty();
+        Specialty urinalysis = new Specialty();
+        Specialty kinesiology = new Specialty();
+        vet.addSpecialty(radiology);
+        vet.addSpecialty(urinalysis);
+        //vet.addSpecialty(kinesiology);
+
+        List<Specialty> testList = vet.getSpecialties();
+        assertThat(testList).isEqualTo(vet.getSpecialties());
```

Comment:
This is passing consistently? This is a similar issue with the setAndGetPetTest in the OwnerTest.java